common.blocks/select/select.en.md

```diff
@@ -377,9 +381,9 @@ Specifies:
 
 Type: `String`.
 
-Specifies content of the button if any menu item is not selected.
+Specifies the button lable if any menu item is not selected.
 
-Use for the button with <a href="#mode-check">type modifier with check value</a> or with <a href="#mode-radiocheck">radio-check</a> value.
+Use for the button when the <a href="#mode-check">type modifier is set check</a> or to <a href="#mode-radiocheck">radio-check</a> value.
```

Comment:
I propose:
Use for the button when the type modifier is set to <a href="#mode-check">check</a> or to <a href="#mode-radiocheck">radio-check</a> value.